plugins/google-cloud/tests/datastore.spec.ts

```diff
@@ -76,14 +76,14 @@ describe('Google Cloud Storage', () => {
           return {
             __esModule: true,
             default: class Foo extends originalModule {
-              datastore: any;
-              constructor(props) {
+              public datastore: object;
+              public constructor(props) {
                 super(props);
                 this.datastore = {
                   key: jest.fn(),
-                  save: _keyData => Promise.resolve([]),
```

Comment:
`_keyData` 🤔 ... I think we need this.

Reply:
Basically, it's unused in this test, so we can avoid its use

Reply:
Nevermind, is a test